src/test/groovy/unit/org/pih/warehouse/core/UserDomainSpec.groovy

```diff
@@ -0,0 +1,47 @@
+package unit.org.pih.warehouse.core
+
+import grails.test.mixin.Mock
+import org.pih.warehouse.core.User
+import spock.lang.Ignore
+import spock.lang.Specification
+
+@Mock(User)
+class UserDomainSpec extends Specification {
```

Comment:
Yeah that was in my mind here, if we have 'Service', 'Controller', then we could have 'Domain' for clarity.

Reply:
I changed package name from 'unit.org.(...)' to 'org.(...)' and now we need to keep this as UserDomainSpec, because we already have UserSpec in package (in src/integraion-test)